### PR TITLE
Allow disabling unsafe implicit conversions in wxString

### PR DESCRIPTION
**Pull request by vadz**

### CODE REVIEW

File: setup.h.in [hunk 1]
```diff
@@ -545,7 +547,7 @@
 #define wxUSE_DRAG_AND_DROP 0
 
 #ifdef __WXMSW__
-#define wxUSE_ACCESSIBILITY 1
+#define wxUSE_ACCESSIBILITY 0
```
Comment (lanurmi) [inline setup.h.in]: Looks like a completely unrelated modification.

Comment (vadz) [inline setup.h.in]: Thanks for noticing this! It looks like this `1` was introduced manually in 8b5387bc3da33f437a53edcb11847802b628eb70 because this was done by regenerating this auto-generated file. This doesn't really matter because `setup.h.in` is only used with configure which will set this symbol correctly anyhow, but I'll separate this change from the other commits to keep things clean.

File: include/wx/msw/setup0.h [hunk 2]
```diff
@@ -1432,9 +1444,9 @@
 // Should wxDC provide SetTransformMatrix() and related methods?
 //
 // Default is 1 but can be set to 0 if this functionality is not used. Notice
-// that currently wxMSW, wxGTK3 support this for wxDC and all platforms support
-// this for wxGCDC so setting this to 0 doesn't change much if neither of these
-// is used (although it will still save a few bytes probably).
+// that currently only wxMSW supports this so setting this to 0 doesn't change
+// much for non-MSW platforms (although it will still save a few bytes
+// probably).
```
Comment (a-wi) [inline include/wx/msw/setup0.h]: Since 49000defcf and 41a2b3e371 SetTransformMatrix() is supported also by wxGCDC so in practice this feature is not supported only by wxGTK2 (among main ports).

Comment (vadz) [inline include/wx/msw/setup0.h]: @a-wi I've replaced this modification with 4cc45797a120d3ac27e5bc2c417faa03da0d016a which ensures that the changes of 49000defcfb11b409d8935126981b14169ee62a3 are not lost during regeneration of the files with `build/update-setup-h`. Does this address your comment?

Comment (a-wi) [inline include/wx/msw/setup0.h]: Yes, it's fine now.
